blocks/avm-report/avm-report-delayed.js

```diff
@@ -0,0 +1,23 @@
+import { fetchPlaceholders } from '../../scripts/lib-franklin.js';
+
+async function initGooglePlacesAPI() {
+  const placeholders = await fetchPlaceholders();
+  const CALLBACK_FN = 'initAvmPlaces';
+  const { mapsApiKey } = placeholders;
+  const script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.async = true;
+  script.defer = true;
+  script.innerHTML = `
+    window.${CALLBACK_FN} = function(){
+      const input = document.querySelector('form input[name="avmaddress"]');
+      const autocomplete = new google.maps.places.Autocomplete(input, {fields:['formatted_address'], types: ['establishment']});
```

Comment:
`establishment` only returns businesses. This is for finding the value of a home. Swap this to `address`